test/extensions/quic_listeners/quiche/envoy_quic_server_session_test.cc

```diff
@@ -70,7 +70,9 @@ class TestEnvoyQuicServerConnection : public EnvoyQuicServerConnection {
     return EnvoyQuicConnection::connectionStats();
   }
 
-  MOCK_METHOD(void, SendConnectionClosePacket, (quic::QuicErrorCode, const std::string&));
+  MOCK_METHOD(void, SendConnectionClosePacket,
+              (quic::QuicErrorCode, quic::QuicIetfTransportErrorCodes ietf_error,
```

Comment:
Nit: Please be consistent here, either always have the argument name spelled, or always not spelled.

Reply:
done